core/chains/evm/gas/block_history_estimator.go

```diff
@@ -340,9 +340,9 @@ func (b *BlockHistoryEstimator) haltBumping(attempts []EvmPriorAttempt) error {
 	}
 	// Return error to prevent bumping if gas price is nil or if EIP1559 is enabled and tip cap is nil
 	if maxGasPrice == nil || (b.eConfig.EIP1559DynamicFees() && maxTipCap == nil) {
-		errorMsg := fmt.Sprintf("%d percentile price is not set. This is likely because there aren't any valid transactions to estimate from. Preventing bumping until valid price is available to compare", percentile)
-		b.logger.Debugf(errorMsg)
-		return errors.New(errorMsg)
+		err := fmt.Errorf("%d percentile price is not set. This is likely because there aren't any valid transactions to estimate from. Preventing bumping until valid price is available to compare", percentile)
+		b.logger.Debugw("Invalid configuration", "err", err)
```

Comment:
I think "Invalid configuration" does't describe this accurately. Maybe to keep the original behavior couldn't we just do 
```suggestion
		b.logger.Debug(err.Error())
```

Reply:
Ah, I misread the first sentence as referring to configuration. I think it is important to follow our standard for errors in structured logs here. What would be a more accurate message?
```suggestion
		b.logger.Debugw("Bumping halted", "err", err)
```

Reply:
I think that's better!